AWSIoT/Internal/AWSIoTKeychain.m

```diff
@@ -350,22 +350,22 @@ + (NSData *)getPrivateKeyBits:(NSString*)tag {
     return (__bridge NSData *)privateKeyBits;
 }
 
-+ (SecIdentityRef)getIdentityRef:(NSString*)tag {
++ (SecIdentityRef)getIdentityRef:(NSString*)privateKeyTag certificateLabel:(NSString *)certificateLabel {
     OSStatus sanityCheck = noErr;
     SecIdentityRef identityRef = NULL;
     
     NSMutableDictionary * queryIdentityRef = [[NSMutableDictionary alloc] init];
     
     [queryIdentityRef setObject:(id)kSecClassIdentity forKey:(id)kSecClass];
-    [queryIdentityRef setObject:tag forKey:(id)kSecAttrApplicationTag]; 
+    [queryIdentityRef setObject:privateKeyTag forKey:(id)kSecAttrApplicationTag];
+    [queryIdentityRef setObject:certificateLabel forKey:(id)kSecAttrLabel];
```

Comment:
This line is the real meat of the change. I had to pass the `certificateLabel` through the call chain so it would be available here.